addons/udes_stock/models/common.py

```diff
@@ -0,0 +1,58 @@
+import re
```

Comment:
I think we're okay with this in 'udes_stock' for now at least. It's something we can keep under review going forwards though.

It may be in future we'd want a similar system in place for other areas, but without that being thought about yet I think it's better to keep this where it as and if we need to we can move it later.